tests/unit/core/test_searchUtils.js

```diff
@@ -223,7 +222,7 @@ describe(__filename, () => {
   });
 
   describe('fixFiltersForAndroidThemes', () => {
-    it.each([ADDON_TYPE_THEMES_FILTER, ADDON_TYPE_THEME])(
+    it.each([ADDON_TYPE_THEME])(
```

Comment:
We don't really need `it.each()` here anymore, it seems.

Reply:
Indeed. I'll remove it in the next iteration.